src/main/java/com/terraforged/mod/server/command/TerraCommand.java

```diff
@@ -247,9 +245,9 @@ private static int findBiome(CommandContext<CommandSource> context) throws Comma
         ServerWorld world = context.getSource().asPlayer().getServerWorld();
         Search search = new BiomeSearchTask(pos, biome, world.getChunkProvider().getChunkGenerator(), getBiomeProvider(context));
         int identifier = doSearch(server, playerID, search);
-        context.getSource().sendFeedback(createIdentifierMessage(identifier)
+        context.getSource().sendFeedback(createPrefix(identifier)
                                          .appendSibling(new StringTextComponent(" Searching for "))
-                                         .appendSibling(createBiomeDisplayMessage(biome))
+                                         .appendSibling(createTitleWithHover(biome.getDisplayName().getFormattedText(), biome.getRegistryName().toString()))
```

Comment:
`biome.getDisplayName()` is client-only so cannot be used. Need to revert to just displaying the registry name